tasks.py

```diff
@@ -24,16 +24,16 @@
 
 
 @task(
-    iterable=["component", "docker_args"],
+    iterable=["component", "docker_arg"],
     help={
         "component": "The components to build - if none given defaults to: "
         + ", ".join(ALL_COMPONENTS),
         "dry_run": "Do not perform any changes, just generate configs and log what would be done",
-        "docker_args": "Arguments to build docker images --docker-args foo=bar. "
-        + "Repeat for multiple build arguments.",
+        "docker_arg": "Arguments to build docker images, e.g. --docker-arg foo=bar "
```

Comment:
How about `Arguments to pass to docker build...`?

Reply:
Fixed